internal/app/summon/command/temp_factory_test.go

```diff
@@ -0,0 +1,99 @@
+package command
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
I thought we made our decision yesterday about this:

> We use std lib `testing` for everything, with the addition of assertion func’s provided by `testify`, and mock’s from `testify` when needed

If that's the decision, we should not go any deeper down the Go convey hole....

Reply:
I implemented this before that decision :/